deploy/crds/policy.open-cluster-management.io_operatorpolicies.yaml

```diff
@@ -216,52 +261,56 @@ spec:
                 - type
                 x-kubernetes-list-type: map
               relatedObjects:
-                description: List of resources processed by the policy
+                description: RelatedObjects reports a list of resources associated
+                  with the operator policy.
                 items:
-                  description: RelatedObject is the list of objects matched by this
-                    Policy resource.
+                  description: RelatedObject contains the details of an object matched
```

Comment:
```suggestion
                  description: RelatedObject contain the details of an object matched
```

Reply:
`RelatedObject` is singular--should this still be "RelatedObject contains" rather than "RelatedObject contain"?